tests/fuzz/fuzz-bigsize.c

```diff
@@ -14,21 +14,22 @@ void run(const uint8_t *data, size_t size)
 	const uint8_t **wire_chunks, *wire_ptr;
 	size_t wire_max;
 
-	wire_chunks = get_chunks(NULL, data, size, 8);
-	for (size_t i = 0; i < tal_count(wire_chunks); i++) {
-		wire_max = tal_count(wire_chunks[i]);
-		wire_ptr = wire_chunks[i];
+	for (size_t max = 1; max <= BIGSIZE_MAX_LEN; max++) {
+		wire_chunks = get_chunks(NULL, data, size, max);
```

Comment:
Nit: `chunk_size` would be a more descriptive name

```suggestion
	for (size_t chunk_size = 1; chunk_size <= BIGSIZE_MAX_LEN; chunk_size++) {
		wire_chunks = get_chunks(NULL, data, size, chunk_size);
```